CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+To contribute to Artipie project you need JDK-11 and Maven 3.2+.
```

Comment:
No reference link is provided.

Reply:
@Sammers21 reference to what? I suppose contributors are able to install Java and Maven by themselves, also there are no standard links for that - each OS distribution has its own package for Java and Maven.

Reply:
@g4s8, for jdk 11 you can refer to https://adoptopenjdk.net/ or https://jdk.java.net/11/. For maven, you can refer to https://maven.apache.org/